<?xml version="1.0" ?>
<launch>
  <arg name="robot_ip" />
  <arg name="load_gripper" default="true" />
  <include file="$(find franka_control)/launch/franka_control.launch" >
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>

  <rosparam command="load" file="$(find franka_example_controllers)/config/franka_example_controllers.yaml" />

  <node name="mouse_teleop" pkg="mouse_teleop" type="mouse_teleop.py" args="--test" respawn="true" />
  <node name="omega7" pkg="omega_ros" type="omega7" args="--test" respawn="true" />
  <node name="phidget" pkg="phidget_ros" type="phidget" args="--test" respawn="true" />
  
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  args="cartesian_pose_example_controller"/>
  <node pkg="rviz" type="rviz" output="screen" name="rviz" args="-d $(find franka_example_controllers)/launch/robot.rviz"/>
</launch>
